scripts/install.js

```diff
@@ -217,6 +254,9 @@ function downloadBinary() {
           .on('error', e => reject(e))
           .on('close', () => resolve());
       }).then(() => {
+        if (process.env.SENTRYCLI_CHECKSUM_VALIDATION !== '1') {
```

Comment:
let's explicitly put skip in so it's clear what it's being used for?

```suggestion
        if (process.env.SENTRYCLI_SKIP_CHECKSUM_VALIDATION !== '1') {
```